x-pack/plugins/uptime/public/components/functional/integration_group.tsx

```diff
@@ -0,0 +1,234 @@
+              message={i18n.translate(
+                'xpack.uptime.monitorList.infraIntegrationAction.container.message',
+                {
+                  defaultMessage: 'Show container metrics',
+                }
+              )}
+              tooltipContent={i18n.translate(
+                'xpack.uptime.monitorList.infraIntegrationAction.docker.tooltip',
+                {
+                  defaultMessage: 'Check Infrastructure UI for container ID "{containerId}"',
+                  values: {
+                    containerId,
+                  },
+                }
+              )}
+            />
+          </EuiFlexItem>
+        </React.Fragment>
+      ) : null}
+      {isLogsAvailable ? (
+        <React.Fragment>
+          <EuiFlexItem>
+            <IntegrationLink
+              ariaLabel={i18n.translate(
+                'xpack.uptime.monitorList.loggingIntegrationAction.kubernetes.ariaLabel',
+                {
+                  defaultMessage: 'Show pod logs',
+                }
+              )}
+              href={getLoggingKubernetesHref(monitor, basePath)}
+              iconType="loggingApp"
+              message={i18n.translate(
+                'xpack.uptime.monitorList.loggingIntegrationAction.kubernetes.message',
+                {
+                  defaultMessage: 'Show pod logs',
+                }
+              )}
+              tooltipContent={i18n.translate(
+                'xpack.uptime.monitorList.loggingIntegrationAction.kubernetes.tooltip',
+                {
+                  defaultMessage: 'Check for logs for pod UID "{podUid}"',
+                  values: {
+                    podUid,
+                  },
+                }
+              )}
+            />
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <IntegrationLink
+              ariaLabel={i18n.translate(
+                'xpack.uptime.monitorList.loggingIntegrationAction.container.id',
+                {
+                  defaultMessage: 'Show container logs',
+                }
+              )}
+              href={getLoggingContainerHref(monitor, basePath)}
+              iconType="loggingApp"
+              message={i18n.translate(
+                'xpack.uptime.monitorList.loggingIntegrationAction.container.message',
+                {
+                  defaultMessage: 'Show container logs',
+                }
+              )}
+              tooltipContent={i18n.translate(
+                'xpack.uptime.monitorList.loggingIntegrationAction.container.tooltip',
+                {
+                  defaultMessage: 'Check Logging UI for container ID "{containerId}"',
+                  values: {
+                    containerId,
+                  },
+                }
+              )}
+            />
+          </EuiFlexItem>
+        </React.Fragment>
+      ) : null}
+    </EuiFlexGroup>
+  ) : (
+    <FormattedMessage
+      defaultMessage="No integrated applications available"
```

Comment:
Thanks for including this default state!

Reply:
++ it's how you know I actually performed a manual test 😆